test/clj/centrebull/test/db/ranges.clj

```diff
@@ -0,0 +1,8 @@
+(ns centrebull.test.db.ranges
+  (:require [clojure.test :refer :all]))
+
+(defn create!
+  [em rv]
+  (fn [range]
+    (is (= range em))
```

Comment:
Swap the order of these values for comparision.

They typical order in clojure (to have it reflect correctly in intelliJ ) on error is : expected then actual